felix/routetable/route_table.go

```diff
@@ -120,7 +122,7 @@ func (t Target) RouteType() int {
 		return syscall.RTN_BLACKHOLE
 	case TargetTypeProhibit:
 		return syscall.RTN_PROHIBIT
-	case TargetTypeUnicast:
+	case TargetTypeUnicastGlobal, TargetTypeUnicastLinkLocal:
```

Comment:
D'oh, think we've already got this in Enterprise but with a different name.  Please can you upstream?

Reply:
https://github.com/projectcalico/calico/pull/6082